es6-shim.js

```diff
@@ -2842,7 +2842,11 @@
         };
 
         MapIterator.prototype = {
+          isMapIterator: true,
```

Comment:
Can't think why anyone would want to forge it, but I also can't think of anything to make it more unforgeable.